apps/web-giddh/src/app/ledger/ledger.module.ts

```
@@ -63,6 +63,7 @@ import { GiddhLedgerPaginatorModule } from '../shared/giddh-ledger-paginator/gid
 import { BankIntegrationModule } from '../shared/bank-integration/bank-integration.module';
 import { CarouselComponent } from '../shared/carousel/carousel.component';
 import { AsideMenuCreateTaxModule } from '../shared/aside-menu-create-tax/aside-menu-create-tax.module';
+import { SalesPersonService } from '../shared/sales-person/utility/sales-person.service';
 import { AsideMenuAccountModule } from '../shared/aside-menu-account/aside.menu.account.module';
 import { SalesPersonService } from '../shared/sales-person/utility/sales-person.service';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SalesPersonService'. Consider to delete it or rename it.

'SalesPersonService' is defined here:


(lint/suspicious/noRedeclare)